data_structures/skip_list.cpp

```diff
@@ -0,0 +1,241 @@
+/**
```

Comment:
use the comment structure as below:
```cpp
/**
 * @file 
 * @brief Add one line description here
 * @details 
 * This is a multi line
 * description containing links, references,
 * math equations, etc in Markdown syntax
 * @author [Name](https://github.com/handle)
 */
```

this is the structure being requested in the comments [above](https://github.com/TheAlgorithms/C-Plus-Plus/pull/941/files#r451690314)

Reply:
I see. I'll change it

Reply:
Done (?)